starboard/common/file_wrapper.h

```diff
@@ -0,0 +1,44 @@
+// Copyright 2024 The Cobalt Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef STARBOARD_COMMON_FILE_WRAPPER_H_
+#define STARBOARD_COMMON_FILE_WRAPPER_H_
+
+typedef struct FileStruct {
```

Comment:
The struct is already defined in starbboard/common/file.h right?

Reply:
Yea there were errors with namespace keyword and extern C usage when I implemented this in starbboard/common/file.h. Removed from the file.